iina/AppDelegate.swift

```diff
@@ -207,12 +207,16 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
   }
 
-  private func showWelcomeWindow() {
+  private func showWelcomeWindow(shouldRefresh: Bool = false) {
```

Comment:
I'd prefer something along the lines of `showWelcomeWindow(checkingForUpdatedData: Bool = false)`.